tests/StandortControllerTest.cs

```diff
@@ -180,7 +186,109 @@ public async Task GetWithEmptyStrings()
+    }
+
+    [TestMethod]
+    public void AddMinimalStandortReturnsCreatedResult()
+    {
+        var newStandort = new Standort
+        {
+            Bezeichnung = TestStandortBezeichnung,
+            UserErstellung = "Marky Mark Tribute Band Member",
+        };
+        var response = controller.Create(newStandort);
+        Assert.IsInstanceOfType(response, typeof(CreatedAtActionResult));
+        controller.Delete(newStandort.Id);
+    }
+
+    [TestMethod]
+    public void AddFullStandortReturnsCreatedResult()
+    {
+        var newStandort = new Standort
+        {
+            Bohrungen = new List<Bohrung>(),
+            Bemerkung = "Mamady Doumbo",
+            Gemeinde = 4321,
+            GrundbuchNr = "hiKbSwsDBTXDyRf",
+            Bezeichnung = TestStandortBezeichnung,
+        };
+        var response = controller.Create(newStandort);
+        Assert.IsInstanceOfType(response, typeof(CreatedAtActionResult));
+        controller.Delete(newStandort.Id);
+    }
+
+    [TestMethod]
+    public void DeleteStandortReturnsOk()
+    {
+        context.Standorte.Add(new Standort
+        {
+            Bemerkung = "Ismaila Kida",
+            Gemeinde = 1234,
+            GrundbuchNr = "cKt6QheQdD7WDjJ",
+            Bezeichnung = TestStandortBezeichnung,
+        });
+        context.SaveChanges();
+        Assert.AreEqual(6001, context.Standorte.Count());
+
+        var standortToDelete = context.Standorte.Single(s => s.Bezeichnung == TestStandortBezeichnung);
+        var response = controller.Delete(standortToDelete.Id);
+
+        Assert.IsInstanceOfType(response, typeof(OkResult));
+        Assert.AreEqual(6000, context.Standorte.Count());
+    }
+
+    [TestMethod]
+    public void TryDeleteInexistentStandortReturnsNotFound()
+    {
+        var controller = new StandortController(context);
+        var response = controller.Delete(1600433);
+
+        Assert.IsInstanceOfType(response, typeof(NotFoundResult));
+        Assert.AreEqual(6000, context.Standorte.Count());
+    }
+
+    [TestMethod]
+    public void EditStandortReturnsOk()
+    {
+        var standortToEdit = context.Standorte.Single(s => s.Id == 31098);
+        var editedBezeichnung = "We love Pasta more than Pesto";
+        standortToEdit.Bezeichnung = editedBezeichnung;
+        var response = controller.Edit(standortToEdit);
+        Assert.AreEqual(context.Standorte.Single(s => s.Id == 31098).Bezeichnung, "We love Pasta more than Pesto");
+        Assert.IsInstanceOfType(response, typeof(OkResult));
+        Assert.AreEqual(6000, context.Standorte.Count());
+    }
+
+    [TestMethod]
+    public void SubmitInvalidEditThrowsException()
+    {
+        var standortToEdit = context.Standorte.Single(s => s.Id == 31099);
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
+        standortToEdit.Bezeichnung = null;
+#pragma warning restore CS8625 // Cannot convert null literal to non-nullable reference type.
```

Comment:
REQ: Anstatt `CS8625` hier abzuschalten kann man hier den _null-forgiving operator_ verwenden:
```suggestion
        standortToEdit.Bezeichnung = null!;
```

Aber: Ich denke `CS8625` wäre ein Kandidat, den wir in den Testprojekten abstellen können. Im `*.csproj`-File des Testprojekts muss `NoWarn` entsprechend erweitert werden:
```xml
<NoWarn>$(NoWarn),CA1014,CS8625</NoWarn>
```